src/auth/auth.service.ts

```diff
@@ -23,6 +23,10 @@ export class AuthService {
     private readonly jwtService: JwtService,
   ) {}
 
+  async verifyJwt(jwtToken: string): Promise<any> {
```

Comment:
change this type to `unknown` or to a specific type if you can determine it, try not to use it `any`

Reply:
you can add interface `AccessTokenPayload`